onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc

```diff
@@ -672,56 +672,57 @@
-        "^test_reduce_log_sum_exp_empty_set_cpu",
-        "^test_reduce_prod_empty_set_cpu"
-    ],
+  "current_failing_tests_pure_DML": [
+    "^test_negative_log_likelihood_loss_input_shape_is_NCd1d2d3_none_no_weight_negative_ignore_index_cpu",
+    "^test_negative_log_likelihood_loss_input_shape_is_NCd1d2d3_none_no_weight_negative_ignore_index_expanded_cpu",
+    "^test_softmax_cross_entropy_input_shape_is_NCd1d2d3_none_no_weight_negative_ignore_index_cpu",
+    "^test_softmax_cross_entropy_input_shape_is_NCd1d2d3_none_no_weight_negative_ignore_index_expanded_cpu",
+    "^test_softmax_cross_entropy_input_shape_is_NCd1d2d3_none_no_weight_negative_ignore_index_log_prob_cpu",
+    "^test_softmax_cross_entropy_input_shape_is_NCd1d2d3_none_no_weight_negative_ignore_index_log_prob_expanded_cpu",
+    "^test_asin_example_cpu",
+    "^test_dynamicquantizelinear_cpu",
+    "^test_dynamicquantizelinear_expanded_cpu",
+    "^test_resize_downsample_scales_linear_cpu",
+    "^test_resize_downsample_sizes_linear_pytorch_half_pixel_cpu",
+    "^test_resize_downsample_sizes_nearest_cpu",
+    "^test_resize_upsample_sizes_nearest_cpu",
+    "^test_roialign_cpu",
+    "^test_dft_axis_cpu",
+    "^test_dft_cpu",
+    "^test_dft_inverse_cpu",
+    // TODO: Remove identity tests when fixed #42638109
+    "^test_identity_cpu",
+    "^test_sequence_map_add_1_sequence_1_tensor_cpu",
+    "^test_sequence_map_add_1_sequence_1_tensor_expanded_cpu",
+    "^test_sequence_map_add_2_sequences_cpu",
+    "^test_sequence_map_add_2_sequences_expanded_cpu",
+    "^test_sequence_map_extract_shapes_cpu",
+    "^test_sequence_map_extract_shapes_expanded_cpu",
+    "^test_sequence_map_identity_1_sequence_1_tensor_cpu",
+    "^test_sequence_map_identity_1_sequence_1_tensor_expanded_cpu",
+    "^test_sequence_map_identity_1_sequence_cpu",
+    "^test_sequence_map_identity_1_sequence_expanded_cpu",
+    "^test_sequence_map_identity_2_sequences_cpu",
+    "^test_sequence_map_identity_2_sequences_expanded_cpu",
+    // onnx 1.15 (opset 20) new and updated op tests (test_affine_grid_???_expanded utilizes ConstantOfShape so it needs to be skipped as well)
+    // https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=1139542&view=logs&j=3032dfba-5baf-5872-0871-2e69cb7f4b6a&t=f0d05deb-fc26-5aaf-e43e-7db2764c07da
+    // only supported with cpu provider
+    "^test_affine_grid_2d",
+    "^test_affine_grid_2d_align_corners",
+    "^test_affine_grid_2d_align_corners_expanded",
+    "^test_affine_grid_2d_expanded",
+    "^test_affine_grid_3d",
+    "^test_affine_grid_3d_align_corners",
+    "^test_affine_grid_3d_align_corners_expanded",
+    "^test_affine_grid_3d_expanded",
+    "^test_constantofshape_float_ones",
+    "^test_constantofshape_int_shape_zero",
+    "^test_constantofshape_int_zeros",
+    "^test_reduce_log_sum_empty_set_cpu",
+    "^test_reduce_log_sum_exp_empty_set_cpu",
+    "^test_reduce_prod_empty_set_cpu",
+    "^test_reduce_min_empty_set_cpu"
```

Comment:
nit: Can we also add a one liner comment on why are we disabling this test? (i.e. [DML EP](https://github.com/microsoft/onnxruntime/blob/2b7e2a5bd07a882a1a1f16e81025a74745ef0394/onnxruntime/core/providers/dml/DmlExecutionProvider/src/ExecutionProvider.cpp#L356) does not execute an operator with empty input tensor because DML does support it.)